assets/scss/components/_form.scss

```diff
@@ -21,3 +26,236 @@
         }
     }
 }
+
+.c-text-input,
+.c-password-input,
+.c-textarea-input {
```

Comment:
I don't want to be pain in the ass, but since the file called `components/form` I'd assume the CSS classes should start with `.c-form...` :D so instead of `c-text-input` it would be better to have `c-form__input` with `c-form__input--text` for text specific styling